src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -314,6 +321,177 @@ export default class GroupComparisonStore extends ComparisonStore {
         },
     });
 
+    readonly mutations = remoteData({
+        await: () => [this.samples, this.mutationEnrichmentProfiles],
+        invoke: async () => {
+            const sampleMolecularIdentifiers = getSampleMolecularIdentifiers(
+                this.samples.result!,
+                this.mutationEnrichmentProfiles.result!
+            );
+            const mutations = await client.fetchMutationsInMultipleMolecularProfilesUsingPOST(
+                {
+                    projection: REQUEST_ARG_ENUM.PROJECTION_DETAILED,
+                    mutationMultipleStudyFilter: {
+                        entrezGeneIds: [
+                            this.selectedMutationMapperGene!.entrezGeneId,
+                        ],
+                        sampleMolecularIdentifiers,
+                    } as MutationMultipleStudyFilter,
+                }
+            );
+            return mutations;
+        },
+    });
+
+    readonly profiledSamplesCount = remoteData({
+        await: () => [this.allSamples, this.coverageInformation],
```

Comment:
@inodb @onursumer  i don't understand why the denominator is all samples here.  that's all samples in ALL studies which are present here.